docs/security-policy-for-authors.md

```diff
@@ -0,0 +1,348 @@
+
+It is important to advise users not to report security
+vulnerabilities in public, e.g.
+
+> Please *do not* use the public issue reporting system on RT or
+> GitHub issues for reporting security vulnerabilities.
+>
+> Please do not disclose the security vulnerability until it has been
+> made public by the maintainers or CPANSec.  That includes blogs,
+> social media, forums, or conferences.
+
+### What to expect for a response
+
+Advise users what to expect in terms of a response.  If there is a
+formal vulnerability workflow then refer to that section of the
+document.
+
+Include a recommendation to report the issue to CPANSec after a
+certain amount of time without a response from the author(s).  For
+example
+
+> The maintainer(s) aim to acknowledge to your security report within
+> 72 hours (three days).  However, this project is maintained by a
+> small group of volunteers in their spare time.  If you have not
+> heard back from the maintainers within 72 hours, then please forward
+> the report to CPANSec at <cpan-security@security.metacpan.org>.
+
+It is reasonable that small projects with a single part time
+maintainer cannot promise a response time.  An alternative might be
+
+> The maintainer(s) aim to acknowledge your security report as soon as
+> possible.  However, this project is maintained by a single person in
+> their spare time, and they cannot guarantee a rapid reponse.  If you
+> have not received a response from the them within a week, then
+> please send a reminder to them and copy the report to CPANSec at
+> <cpan-security@security.metacpan.org>.
+
+Also add a note about what the initial response:
+
+> Note that the initial response to your report will be an
+> acknowledgement, with a possible query for more information.  It
+> will not necessarily include any fixes for the issue.
+
+Also note on which conditions you may forward the security issue to
+other people.  For example,
+
+> The project maintainer(s) may forward this issue to the security
+> contacts for other projects where we believe it is relevant.  This
+> may include embedded libraries or prerequisite modules or system
+> libraries, or downstream software that uses this software.
+>
+> They may also forward this issue to CPANSec if they have not
+> already been copied on this.
+
+## What Software this Policy Applies to
+
+This section discusses what software this applies to, and what are
+considered security issues in the software, and what are not
+considered security issues.  For example,
+
+> Any security vulnerabilities in Foo-Bar are covered by this policy.
+>
+> Security vulnerabilities are considered anything that allows users
+> to execute unauthorised code, access unahtorised resources, or to
+> have an adverse impact on accessibility or performance of a system.
+>
+> Security vulnerabilities in upstream software (embedded libraries,
+> prerequisite modules or system libraries, or in Perl), are not
+> convered by this policy.
```

Comment:
They might be if a dist ships a copy of a C library like Net::AMQP::RabbitMQ does it (to my knowledge).

Reply:
It mentions that in a "However" a couple of paragraphs later. But maybe some rewording is needed.

Reply:
Updated in e28a0b8